### DISCUSSION TITLE
无法自动认证，请填写本地服务器的登录信息！

### DISCUSSION THREAD

**Discussion by git-Ander** [1Panel-dev/1Panel#5286] [Q&A]
终端  无法自动认证，请填写本地服务器的登录信息！  要怎么设置, 服务器地址我也设置了
![image](https://github.com/1Panel-dev/1Panel/assets/12371291/33f85852-92e3-4ddc-be9b-a819d475a18a)
![image](https://github.com/1Panel-dev/1Panel/assets/12371291/4e11fb11-c1e2-4b5b-97f6-e3f8660d9a73)

**Comment by wanghe-fit2cloud** [comment]
首先要确保 root 用户密码正确，然后在1Panel服务器终端使用 `ssh root@127.0.0.1` 命令看是否能登录成功。

**Comment by ZACKLZ** [comment]
我也有这样的问题，我可以用其他工具登录ssh，1panel就是没法用

**Reply by wanghe-fit2cloud** [reply]
- 首先验证用户密码方式是否正常，如图：
![image](https://github.com/user-attachments/assets/74e3b4ec-ea0b-4d61-a340-88bd8670b4a4)


- SSH 密钥登录，你需要先将公钥写入远程服务器的指定文件，然后将私钥信息添加至 1Panel。如果有问题，就先手动 ssh 命令验证一下是否可用。

> 本地未能复现上述问题。

**Reply by ZACKLZ** [reply]
> * 首先验证用户密码方式是否正常，如图：
>   ![image](https://private-user-images.githubusercontent.com/46734380/351182820-74e3b4ec-ea0b-4d61-a340-88bd8670b4a4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE3MDkzNzIsIm5iZiI6MTcyMTcwOTA3MiwicGF0aCI6Ii80NjczNDM4MC8zNTExODI4MjAtNzRlM2I0ZWMtZWEwYi00ZDYxLWEzNDAtODhiZDg2NzBiNGE0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzIzVDA0MzExMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFhOTViMDU4YzY3ZDg4NWU2MTYwN2JkY2Y0ZTdkNmNjOGRmMDliMDBhYzY0N2NjZDRlMTVjOGMyNmRmNGVjZDkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.a3R9XIagxVOOWW8GpoorR9truF0rvyjxjSxNC5IkASg)
> * SSH 密钥登录，你需要先将公钥写入远程服务器的指定文件，然后将私钥信息添加至 1Panel。如果有问题，就先手动 ssh 命令验证一下是否可用。
> 
> > 本地未能复现上述问题。

为何已经可以用其他工具登录终端还需要这样验证呢？

**Comment by imxood** [comment]
我是这样处理的:

系统是 debian 12, 执行 `ping 127.0.0.1`, 结果超时了, 于是 查看lo接口状态 `sudo ip addr show lo`, 发现其输出, 没有127.0.0.1

于是手动添加 `sudo ip addr add 127.0.0.1/8 dev lo`,

再次执行 `ping 127.0.0.1`, 就能ping通了. 于是在"添加主机"中配好127.0.0.1后, 这个issue中 无法自动认证的问题, 就解决了.

**Comment by wxjer** [comment]
sudo ip addr show lo 有127.0.0.1,发现sshd配置开启了root验证，但关闭了密码验证，打开就好了

**Comment by reaherxiebly** [comment]
是权限的问题，~/.ssh的权限要设置为0700，里面的authorized_keys设置为0600就可以了
